src/consensus/amount.h

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
 // Copyright (c) 2009-2021 The Bitcoin Core developers
+// Copyright (c) 2014-2023 The Vertcoin Core developers
```

Comment:
I don't know when this was added but I don't see any reason to include our copyright in this file.

Reply:
At one point I was wanting to add the Vertcoin copyright tag to all the files added/modified for Vertcoin Core.  With the thought being that this is helpful for any future rebase to know what was added/modified... I then realized that is a bad idea and the Vertcoin copyright should only be on files where new, copyright eligible, code developed for Vertcoin is implemented.  I will remove this copyright comment.